.circleci/config.yml

```diff
@@ -768,18 +768,18 @@ workflows:
           chain-id: 42161
           provider-url: https://arbitrum-mainnet.infura.io/v3/$INFURA_API_KEY
 
-      - fork-test:
+      - fork-test-new:
           name: test-snaxchain-testnet
           toml: omnibus-snaxchain-testnet.toml
-          package: "synthetix-omnibus:latest"
+          # package: "synthetix-omnibus:latest"
```

Comment:
we should remove this commented out line and the one below right?

Reply:
no. after initial deployment - uncomment and change workflow from `fork-test-new` to `fork-test`